UnityProject/Assets/Scripts/Chat/Chat.cs

```diff
@@ -82,6 +84,13 @@ public static void AddChatMsgToChat(ConnectedPlayer sentByPlayer, string message
 			}
 
 			Instance.addChatLogServer.Invoke(chatEvent);
+
+			//Sends OOC message to a discord webhook
+			if (!string.IsNullOrEmpty(ServerData.ServerConfig.DiscordWebhookOOCURL))
```

Comment:
Potentially pull this null check into the SendWebHookMessage - see the commet on DiscordWebhook.cs

Reply:
Added.